src/api/domain/orders-api.ts

```diff
@@ -51,18 +51,15 @@ export const OrdersApiAxiosParamCreator = function (configuration?: Configuratio
         /**
          * Cancel an order
          * @summary Cancel an order
-         * @param {string} xImxEthAddress eth address
-         * @param {string} xImxEthSignature eth signature
          * @param {string} id Order ID to cancel
          * @param {CancelOrderRequest} cancelOrderRequest cancel an order
+         * @param {string} [xImxEthAddress] eth address
+         * @param {string} [xImxEthSignature] eth signature
+         * @param {string} [authorization] Authorization header
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        cancelOrder: async (xImxEthAddress: string, xImxEthSignature: string, id: string, cancelOrderRequest: CancelOrderRequest, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
-            // verify required parameter 'xImxEthAddress' is not null or undefined
-            assertParamExists('cancelOrder', 'xImxEthAddress', xImxEthAddress)
-            // verify required parameter 'xImxEthSignature' is not null or undefined
-            assertParamExists('cancelOrder', 'xImxEthSignature', xImxEthSignature)
+        cancelOrder: async (id: string, cancelOrderRequest: CancelOrderRequest, xImxEthAddress?: string, xImxEthSignature?: string, authorization?: string, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
@carmen0208 introducing the `authorization` param here is a breaking change. We'd need to handle this at the parent workflow.

Reply:
How is the breaking change? it is optional?

Reply:
@cajames I got what you mean, but the **entrypoint** of the workflow should be in this line: https://github.com/immutable/imx-core-sdk/pull/289/files#diff-2a850f6705a84b936769ce7b1231d25e04b6169b5005a64b31caaa02e16ad6d6R1044